test/mpi/distributed/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-ginkgo_create_common_and_reference_test(matrix MPI_SIZE 3)
-ginkgo_create_common_and_reference_test(vector MPI_SIZE 3)
+ginkgo_create_common_and_reference_test(matrix MPI_SIZE 3 DISABLE_EXECUTORS dpcpp)
+ginkgo_create_common_and_reference_test(vector MPI_SIZE 3 DISABLE_EXECUTORS dpcpp)
```

Comment:
I already disabled tests which don't have their kernels implemented directly in the files, is that not enough anymore?

Reply:
*something* broke, I don't remember what it was. I'll check tomorrow

Reply:
`matrix_dpcpp` fails because `build_starting_indices` is not implemented

Reply:
If we want to keep at least part of the test, we could build the partition on `ref` instead of `exec`.

Reply:
Maybe we can do that in a subsequent PR, since it requires more significant changes?